app/controllers/application_controller/buttons.rb

```diff
@@ -246,7 +246,7 @@ def ab_group_delete
 
   private
 
-  BASE_MODEL_EXPLORER_CLASSES = [Vm, MiqTemplate, Service].freeze
+  BASE_MODEL_EXPLORER_CLASSES = [MiqGroup, MiqTemplate, Service, Switch, Tenant, User, Vm].freeze
```

Comment:
Switch?

Reply:
Compute -> Infrastructure -> Networking has `Switch` as its model and it's an explorer screen.